dali/test/python/test_operator_arithmetic_ops.py

```diff
@@ -665,28 +667,31 @@ def check_raises_te(kinds, types, op, shape, _):
                    ((lambda x, y: x ** y), "**")]
 
 def test_bool_disallowed():
+    error_msg = "[Ii]nput[s]? to arithmetic operator `[\S]*` cannot be [a]?[ ]?boolean[s]?. Consider using bitwise operator[s]?"
```

Comment:
nitpick: raises is not case sensitive by default

Reply:
I didn't know. Fixed.